src/styles/utils/fetchClient.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Avoid using 'eslint-disable'. It is generally a better practice to follow the linter's rules as it helps in maintaining the code quality and standards set by the community. In this case, rather than using 'any' for 'data', consider defining an interface that closely matches the structure of the data or use 'unknown' if the structure is not known.